main.js

```diff
@@ -113,18 +113,22 @@ function savePalette() {
 }
 
 function isPaletteUnique(palettesList, singlePalette) {
-    for (i = 0; i < palettesList.length; i++) {
-        var matches = true;
-        for (j = 0; j < palettesList[i].length; j++) {
-            if (palettesList[i][j] !== singlePalette[j]) {
-                matches = false;
-                break;
-            }
-        }
-        if (matches) {
+    for (var i = 0; i < palettesList.length; i++) {
+        console.log('isPaletteUnique', i);
+        if (areArraysEquivalent(palettesList[i], singlePalette)) {
+            return false;
+        } 
+    }
+    return true;   
+}    
+
+function areArraysEquivalent(array1, array2) {
```

Comment:
we should make the parameters more descriptive. Maybe array1 = paletteToCheck & array2 = currentPalette?